src/resources/postcss/views/month/_calendar-event.pcss

```diff
@@ -63,6 +63,28 @@
 		}
 	}
 
+	.tribe-events-calendar-month__day--past {
+
+		.tribe-events-calendar-month__calendar-event--featured {
+
+			&:before {
+				background: var(--color-past-events);
+			}
+		}
+
+		.tribe-common-c-image__bg {
+			filter: grayscale(100%);
```

Comment:
This property won't work on IE11. We need to think of something that'll support IE11.